manifests/install.pp

```diff
@@ -57,9 +57,9 @@
   case $facts['os']['family'] {
     'RedHat': {
       if $manage_repo {
-        yumrepo { 'bintray-rundeck':
+        yumrepo { 'rundeck':
```

Comment:
Renaming the repo might be an issue if unmanaged repos are not purged.  But since the old location seems to be unfetchable, this should not harm too much (I guess):

* users who purge unmanaged repo will not see anything;
* users who keep unmanaged repos will have errors when updating the available packages because some repository are broken, but the actual packages should be found.

Reply:
FWIW, the wholes packagecloud repos are a huge mess and tweaking parts of how they are managed does not make much sense I think.  I refrained from dropping all tuning parameters other than `$manage_repo`, but I doubt anyone can make something useful out of it…